internal/serverinstall/ecs.go

```diff
@@ -0,0 +1,2190 @@
+package serverinstall
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/aws/aws-sdk-go/service/efs"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/service/resourcegroups"
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/waypoint-plugin-sdk/component"
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/builtin/aws/utils"
+	"github.com/hashicorp/waypoint/internal/clicontext"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+	"github.com/hashicorp/waypoint/internal/serverconfig"
+)
+
+const (
+	defaultRunnerLogGroup = "waypoint-runner-logs"
+	defaultServerLogGroup = "waypoint-server-logs"
+
+	defaultTaskCPU     = 512
+	defaultTaskMemory  = 1024
+	defaultTaskFamily  = "waypoint-server"
+	defaultTaskRuntime = "FARGATE"
+
+	defaultSecurityGroupName = "waypoint-server-security-group"
+	defaultNLBName           = "waypoint-server-nlb"
+)
+
+var ulid string
+
+func init() {
+	// generate a ULID for use in FileSystem creation and unique tag value for
+	// this installation
+	ulid, _ = component.Id()
+}
```

Comment:
This feels a bit odd, could we move this inside Install and have it be local instead of global?

Reply:
As we discussed, I removed the `ulid` value completely and changed how we do tags in bcc87175